pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -1,79 +1,180 @@
 package resources_test
 
 import (
-	"fmt"
 	"testing"
```

Comment:
please add a test for migration from v091 to v092 (resource created in 091, provisioner_name -> run_as_role); the test is necessary so we know, what to recommend to our users (because currently the migration notes are not guiding with this change); maybe some state migratory should be introduced, based on the outcome of the test, we can discuss when you have it

Reply:
Done.